lib/src/app/app.dart

```diff
@@ -109,4 +109,34 @@ class AppClient {
         .map((e) => Permission.values.firstWhere((element) => element.value == e))
         .toList();
   }
+
+  Future<ListOrganizationMembersResponse> listOrganizationMembers(Organization org) async {
```

Comment:
i dislike having the proto definition be the return type here, but otherwise i'd be creating a new wrapper which is essentially the same thing. or alternatively two methods: `List<OrganizationMembers> listOrganizationMembers` & `List<OrganizationInvites> listOrganizationInvites` but that doesn't match the proto. I didn't like either of those ideas more than simply just returning the proto type. Open to changing the approach here though.

Reply:
Ya I think this is fine. I think reading protos is actually not that big of a deal (creating protos is more annoying, but we abstract that away from the user)

Reply:
sweet. thanks!